apps/tidl_deploy/NeoTvmCodeGen.py

```diff
@@ -0,0 +1,239 @@
+"""

```

Comment:
I am not quite sure about the policy of the license here since TVM is Apache license.